lisa/tools/lspci.py

```diff
@@ -78,43 +116,28 @@ def __str__(self) -> str:
             f"vendor: {self.vendor}, "
             f"info: {self.device_info}, "
             f"vendor_id: {self.vendor_id}, "
-            f"device_id: {self.device_id}"
+            f"device_id: {self.device_id}, "
+            f"controller_id: {self.controller_id} "
         )
 
-    def parse(self, raw_str: str) -> None:
-        matched_pci_device_info_list = find_groups_in_lines(
-            lines=raw_str,
-            pattern=PATTERN_PCI_DEVICE,
-        )
-        if matched_pci_device_info_list:
-            matched_pci_device_info = matched_pci_device_info_list[0]
-            self.slot = matched_pci_device_info.get("slot", "").strip()
+    def parse(self, raw_str: str, pci_ids: Dict[str, Any]) -> None:
```

Comment:
This method looks not need to be changed, the filter happens on upper caller too. It doesn't need to filter in this method.

Reply:
Fixed it.